app/views/candidate_interface/restructured_work_history/job/_form.html.erb

```diff
@@ -7,11 +7,13 @@
 <% end %>
 
 <div class="app-work-experience__start-date" data-qa="start-date">
-  <%= f.govuk_date_field :start_date, omit_day: true, legend: { text: t('application_form.restructured_work_history.start_date.label'), size: 'm' }, hint: { text: t('application_form.restructured_work_history.start_date.hint_text') } %>
-  <div class="govuk-form-group">
-    <%= f.hidden_field :start_date_unknown, value: false %>
-    <%= f.govuk_check_box :start_date_unknown, true, multiple: false, label: { text: t('application_form.restructured_work_history.start_date_unknown_checkbox') } %>
-  </div>
+  <%= f.govuk_fieldset legend: { text: t('application_form.restructured_work_history.start_date.label'), size: 'm' } do %>
+    <%= f.govuk_date_field :start_date, omit_day: true, legend: nil, hint: { text: t('application_form.restructured_work_history.start_date.hint_text') } %>
```

Comment:
```suggestion
    <%= f.govuk_date_field :start_date, omit_day: true, hint: { text: t('application_form.restructured_work_history.start_date.hint_text') } %>
```

Reply:
I had to explicitly set it to nil, otherwise it'll render an additional legend, defaulting to the attribute value

Reply:
apparently only omitted if nil